### ISSUE TITLE
Restore DB

### ISSUE THREAD

**Issue by cchristelis** [kartoza/feti#559]
# Problem

When I restore the DB and then run `make rebuildindex`, it fails.

# Proposed Solution

Update and fix.

**Comment by anitanh** [comment]
hi @cchristelis have you tried using this DB? 
I am using this DB and it is working both for `make rebuildindex` and for `make rebuildindex-devweb`

https://drive.google.com/open?id=1PLSoagS-Fr45vvRDbleeSEy-QpyAua9u

**Comment by pierrealixt** [comment]
So I used your DB @ann26 and it still wasnt working when I ran `make dbrestore`. At some point, the script hangs...
I had to convert the .dmp to .sql with 
`$> pg_restore latest.dmp > latest.sql` 
and load it with 
`$> psql -d gis < /backups/latest.sql`
Then
`$> make rebuildindex`
worked fine.

@ann26 what OS are you on?

**Comment by anitanh** [comment]
@pierrealixt hmm.. usually what you need to do is put that DB in backups folder and rename it to be `latest.dmp` then do `make dbrestore` then `make rebuildindex` and `make rebuildindex-devweb`

I am using Ubuntu 16.04 LTS

**Comment by pierrealixt** [comment]
@ann26 on my mac (10.12), running `make dbrestore` never worked! 
 ¯\_(ツ)_/¯

**Comment by anitanh** [comment]
@pierrealixt hmm.. It should be working on mac too. @dimasciput also using mac and this run on his machine.
What did you get when you run `make dbrestore` ? You did it in `deployment` folder right?

**Comment by sonlinux** [comment]
Hi @pierrealixt 
For test purposes try to restore from this db version, its not the latest but it should work just fine I restored from it last night, the latest dump works too but just for testing on your side you can start with this.

https://drive.google.com/open?id=1SOSdEjO8U16PugF6Fzk8UFEAs-QAHcm7

Like @ann26 advised place it in the *deployment/backups/* directory and run the restore directive commands

**Comment by pierrealixt** [comment]
@ann26 the script starts like it's working and then it stops... for more than 10 minutes, nothing happen. I had to Ctrl+C
Yes the latest.dmp was in `deployment/backups` and yes I ran the commands in the `deployment` folder. 
We had the same issue with Tim on the testing server.
Anyway I made it working by converting the dump in sql and load it with psql. All good, thanks.

@sonlinux I'll try with this db version and let you know.
